randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -216,6 +218,26 @@ def _encode_hints(self) -> list[dict]:
             for logbook_node in self.game.world_list.all_nodes
             if isinstance(logbook_node, LogbookNode)
         ]
+    
+    def _static_text_changes(self) -> dict[str, str]:
+        full_hash = f"{self.description.shareable_word_hash} ({self.description.shareable_hash})"
+        text = {}
+        difficulty_labels = {
+            "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_HARD_UNLOCKED",
+            "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_NORMAL",
+            "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EASY",
+            "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EXPERT"
+        }
+        for difficulty in difficulty_labels:
+            text[difficulty] = full_hash
+        
+        text["GUI_COMPANY_TITLE_SCREEN"] = "|".join([
+            f"Randovania {VERSION} - open-dread-rando {open_dread_rando_version}",
```

Comment:
Both versions should be taken during the `GameExport` step, to ensure they're using the actual client version instead of the server's.